### PR TITLE
Task: Add FAQLayout component - issue #15

### PR DESCRIPTION
**Pull request by willcary**
Check for correct use of composition, styling, and general compliance with project guidelines.

### CODE REVIEW

File: src/components/FAQLayout/FAQLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import styles from './styles.module.css'
+
+const FAQLayout = (props) => {
```
Comment (wijohnst) [inline src/components/FAQLayout/FAQLayout.jsx]: NIT: Destructure props

`({children})`

File: src/components/FAQLayout/FAQLayout.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	return (
+		<div className={styles[`faq-layout`]}>
+            {props.children}
```
Comment (wijohnst) [inline src/components/FAQLayout/FAQLayout.jsx]: NIT: Indent

**Review by wijohnst — COMMENTED** [review]
Great work! Left a couple of small comments.

**Review by wijohnst — APPROVED** [review]
LGTM! 🚢

**Comment by wijohnst** [comment]
Closes #15

**Comment by willcary** [comment]
Closes #15

**Pull request merged**